modules/services/emacs.nix

```
@@ -81,6 +81,14 @@ in {
       enable = mkEnableOption "systemd socket activation for the Emacs service";
     };
 
+    startWithUserSession = lib.mkOption {
+      default = !cfg.socketActivation.enable;
+      example = cfg.socketActivation.enable;
```

Cannot make manual depend on the configuration:
```suggestion
      example = true;
```

---

fixed